docs/standards.md

```diff
@@ -0,0 +1,167 @@
+## SG2 Standards Checklist
+
+Since SG2 is a living design library that is home to reusable patterns, it's important for all contributing developers to attempt to create patterns the same way. 
+
+Any new work contributed into SG2 must meet the following standards:
+
+### PatternLab Organization
+
+#### How patterns are organized:
+- Patterns should be listed alphabetically in the pattern list. No numbers (e.g. `atoms/forms/text-input` instead of `00-atoms/10-forms/05-text`). 
+- Talk about where SCSS and JS files should go (in a separate .scss directory?), as well as Twig and .md files (no subdirectories?). We may still need to make a decision about this.
+
+##### Atoms
+- An atom should be the basic HTML elements.
+- They can sometimes be more than one element (e.g. an unordered list).
+
+##### Molecules
+- A molecule should include most of the markup and pull in atoms as necessary.
+- Not every element in a molecule needs to be an atom.
+
+##### Organisms
+- Organisms should be very little markup, it should be mostly wrappers around molecules or atoms.
+
+##### Templates
+- Layouts should be used for templates but not molecules
```

Comment:
My understanding is that Templates will be used for the general layouts (e.g. regions of the page.)

Reply:
Yeah, that's what we've decided on. I think that's good language to put in there. This whole section should be revised to sound a little more eloquent. It would also be good to add examples for each type of thing.